src/App.jsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import './App.scss';
+import Person from './components/Person/Person';
```

Comment:
fix the tests, check how this component is imported there and adjust your code to that
<img width="548" alt="image" src="https://user-images.githubusercontent.com/71132144/235624030-83e45dd9-39d1-4622-bf28-d212fc4525a8.png">